src/EditorFeatures/CSharpTest/Diagnostics/SimplifyTypeNames/SimplifyTypeNamesTests.cs

```diff
@@ -3755,7 +3755,6 @@ void M()
     }
 }",
                 options: Option(CodeStyleOptions.QualifyPropertyAccess, false, NotificationOption.Warning),
-                diagnosticCount: 1,
```

Comment:
It should assert that the size equals one, so that we're not getting a failure from Single throwing an exception, but rather from an explicit assertion that there is only one diagnostic.